src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/ProfessionalUserInternalController.java

```diff
@@ -1,196 +1,196 @@
-    @Secured("prd-admin")
-    public ResponseEntity<ModifyUserRolesResponse> modifyRolesForExistingUserOfOrganisation(
-            @RequestBody UserProfileUpdatedData userProfileUpdatedData,
-            @Pattern(regexp = ORGANISATION_IDENTIFIER_FORMAT_REGEX, message = ORG_ID_VALIDATION_ERROR_MESSAGE) @PathVariable("orgId") String orgId,
-            @PathVariable("userId") String userId,
-            @RequestParam(name = "origin", required = false, defaultValue = "EXUI") Optional<String> origin
-    ) {
-
-        organisationIdentifierValidatorImpl.validateOrganisationExistsWithGivenOrgId(orgId);
-
-        //Received request to update user roles of an organisation
-        return modifyRolesForUserOfOrganisation(userProfileUpdatedData, userId, origin);
-
-    }
-}
+package uk.gov.hmcts.reform.professionalapi.controller.internal;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import static uk.gov.hmcts.reform.professionalapi.controller.constants.ProfessionalApiConstants.EMPTY;
+import static uk.gov.hmcts.reform.professionalapi.controller.constants.ProfessionalApiConstants.ORGANISATION_IDENTIFIER_FORMAT_REGEX;
+import static uk.gov.hmcts.reform.professionalapi.controller.constants.ProfessionalApiConstants.ORG_ID_VALIDATION_ERROR_MESSAGE;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.Authorization;
+
+import java.util.Optional;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.professionalapi.controller.SuperController;
+import uk.gov.hmcts.reform.professionalapi.controller.response.ProfessionalUsersResponse;
+import uk.gov.hmcts.reform.professionalapi.domain.ModifyUserRolesResponse;
+import uk.gov.hmcts.reform.professionalapi.domain.UserProfileUpdatedData;
+
+
+@RequestMapping(
+        path = "refdata/internal/v1/organisations",
+        produces = APPLICATION_JSON_VALUE
+)
+@Validated
+@RestController
+@Slf4j
+public class ProfessionalUserInternalController extends SuperController {
+
+    @ApiOperation(
```

Comment:
why entire class is showing difference here.could you please revert the file and add only the changes that you modified

Reply:
Please can you apply hidden whitespaces setting and review. I have tried by reverting the change but no effect. I will take care this next time. Thanks.